### DISCUSSION TITLE
NoM Phase 1 - Forum discussion post

### DISCUSSION THREAD

**Discussion by sugoibtc** [coinselor/twitter#6] [Tweet Ideas]
first draft (does not fit in the character limit on X yet)

The road to NoM - Phase 1

Gear up, as the Zenon Network is on the path to revolutionize the blockchain landscape by creating a self-sustaining network.

- A balance of ASIC and CPU-friendly PoW,
- Integrating a decentralized, peer-to-peer mining pool for BTC & RandomX
- Enhancing censorship-resistance and decentralization.

https://forum.hypercore.one/t/network-of-momentum-phase-1/253

**Comment by coinselor** [comment]
It's really hard to sum up these threads in a single tweet given our character limitation, we could try to create a thread with multiple replies. 

I think the easiest would be just to highlight a single "feature" of Phase 1 per tweet. 

Another consideration would be to include the CTA to join the conversation/find out more in a reply instead that in the main tweet, as linking to a website outside of twitter/X hurts performance.
